### PR TITLE
[gitpod] configure python version

### PR DESCRIPTION
**Pull request by akosyakov**
#### What it does

Gitpod CI were failing for PRs from forks because yarn could not find python. 
This PR configure python for npm/yarn. 

#### How to test

This PR is opened from a fork. Gitpod CI should succeed for it.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: .gitpod.dockerfile [hunk 1]
```diff
@@ -19,5 +19,7 @@ RUN bash -c ". .nvm/nvm.sh \
     && nvm install $NODE_VERSION \
     && nvm use $NODE_VERSION \
     && nvm alias default $NODE_VERSION \
+    && npm config set python /usr/bin/python --global \
+    && npm config set python /usr/bin/python \
     && npm install -g yarn"
```
Comment (jankeromnes) [inline .gitpod.dockerfile:24]: Gitpod's default image also installs `typescript` here -- not needed for Theia?

https://github.com/gitpod-io/workspace-images/blob/23a8109ab27b1355421477366657ce85b54b8043/full/Dockerfile#L196

Comment (akosyakov) [reply .gitpod.dockerfile:24]: we already have typescript in Theia package.json that should do

**Comment by jankeromnes** [comment]
Opening this PR in Gitpod yields a strange error:

```
gitpod /workspace/theia $ {
> yarn
> }; exit
yarn install v1.22.4
$ node-gyp install
gyp info it worked if it ends with ok
gyp info using node-gyp@5.0.5
gyp info using node@10.19.0 | linux | x64
gyp http GET https://nodejs.org/download/release/v10.19.0/node-v10.19.0-headers.tar.gz
gyp http 200 https://nodejs.org/download/release/v10.19.0/node-v10.19.0-headers.tar.gz
gyp http GET https://nodejs.org/download/release/v10.19.0/SHASUMS256.txt
gyp http 200 https://nodejs.org/download/release/v10.19.0/SHASUMS256.txt
10.19.0
gyp info ok 
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
warning url-loader@1.1.2: Invalid bin field for "url-loader".
info fsevents@2.1.2: The platform "linux" is incompatible with this module.
info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
warning monaco-languageclient@0.13.0: The engine "vscode" appears to be invalid.
warning vscode-languageclient@6.1.0: The engine "vscode" appears to be invalid.
info fsevents@1.2.11: The platform "linux" is incompatible with this module.
info "fsevents@1.2.11" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > chai-string@1.5.0" has unmet peer dependency "chai@^4.1.2".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/application-manager > font-awesome-webpack@0.0.5-beta.2" has unmet peer dependency "font-awesome@>=4.3.0".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/cli > requestretry@3.1.0" has unmet peer dependency "request@2.*.*".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/messages > react-perfect-scrollbar@1.5.3" has unmet peer dependency "react@>=16.3.3".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/messages > react-perfect-scrollbar@1.5.3" has unmet peer dependency "react-dom@>=16.3.3".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/scm > react-autosize-textarea@7.0.0" has unmet peer dependency "react@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/scm > react-autosize-textarea@7.0.0" has unmet peer dependency "react-dom@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "workspace-aggregator-03d5b99b-da8f-4fc1-99be-ddc0379e530e > @theia/application-manager > font-awesome-webpack > less-loader@2.2.3" has incorrect peer dependency "less@^2.3.1".
warning Workspaces can only be enabled in private projects.
[5/5] Building fresh packages...
[10/13] ⢀ oniguruma
[7/13] ⢀ nsfw
[9/13] ⢀ drivelist
[4/13] ⢀ puppeteer
error /workspace/theia/node_modules/@theia/electron: Command failed.
Exit code: 127
Command: node scripts/post-install.js
Arguments: 
Directory: /workspace/theia/node_modules/@theia/electron
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@10.19.0 | linux | x64
gyp ERR! configure error 
gyp ERR! stack Error: Can't find Python executable "/usr/bin/python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (/workspace/theia/node_modules/node-gyp/lib/configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/workspace/theia/node_modules/node-gyp/lib/configure.js:406:16)
gyp ERR! stack     at F (/workspace/theia/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/workspace/theia/node_modules/which/which.js:80:29)
gyp ERR! stack     at /workspace/theia/node_modules/which/which.js:89:16
gyp ERR! stack     at /workspace/theia/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /workspace/theia/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:153:21)
gyp ERR! System Linux 4.14.138+
gyp ERR! command "/home/gitpod/.nvm/versions/node/v10.19.0/bin/node" "/workspace/theia/dev-packages/electron/node-gyp-cli.js" "rebuild"
gyp ERR! cwd /workspace/theia/dev-packages/electron/native
gyp ERR! node -v v10.19.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
Error: "../node-gyp-cli.js" exited with 1
    at ChildProcess.subprocess.once (/workspace/theia/dev-packages/electron/scripts/post-install.js:36:40)
    at Object.onceWrapper (events.js:286:20)





🍌 This task ran as part of a workspace prebuild.
🎉 You just saved a minute of watching your code build.


[1] 1575
yarn run v1.22.4
$ theia start --plugins=local-dir:../../plugins ../.. --hostname=0.0.0.0
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '../lib/theia'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/workspace/theia/dev-packages/cli/bin/theia:2:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by akosyakov** [comment]
Using `$(which python)` fixed the issue.

**Pull request closed without merging**